appveyor.yml

```diff
@@ -1,9 +1,17 @@
 environment:
   matrix:
-  - JULIAVERSION: "julialang/bin/winnt/x86/0.6/julia-0.6-latest-win32.exe"
-  - JULIAVERSION: "julialang/bin/winnt/x64/0.6/julia-0.6-latest-win64.exe"
-  - JULIAVERSION: "julianightlies/bin/winnt/x86/julia-latest-win32.exe"
-  - JULIAVERSION: "julianightlies/bin/winnt/x64/julia-latest-win64.exe"
+  - julia_version: 1.0
```

Comment:
In DataFrames, we're moving to test windows via Travis and getting rid of appveyor all together; it'd be nice to follow a similar pattern here: https://github.com/JuliaData/DataFrames.jl/blob/master/.travis.yml